src/material-experimental/mdc-button-toggle/harness/button-toggle-harness.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk-experimental/testing';
+import {coerceBooleanProperty} from '@angular/cdk/coercion';
+import {ButtonToggleHarnessFilters} from './button-toggle-harness-filters';
+
+/**
+ * Harness for interacting with a mat-button-toggle in tests.
+ * @dynamic
+ */
+export class MatButtonToggleHarness extends ComponentHarness {
```

Comment:
@mmalerba open question here: his harness doesn't allow us to interact with the group that can be around a button toggle. Where would that functionality go? Putting it on the one for the button toggle feels weird, because it won't always be in a group so the alternative is to have a dedicated harness for the button group. That also feels a little weird to me, because you won't have access to the functionality on the toggle harness which defeats the purpose a little bit.

Reply:
Can't the group harness just expose methods to get harnesses for the individual toggles? (e.g. `getActiveButtonToggle()`, `getAllButtonToggles()`)

Reply:
It can, but I wasn't sure whether we'd want to expose more than one harness per entry point.

Reply:
Yeah I think its fine to have multiple harness classes in cases where it makes sense. This seems like a good case for it

Reply:
FWIW: I'm doing similar things to what Miles proposed for the radio-button and radio-group.